style.css

```diff
@@ -0,0 +1,146 @@
+body {
+  background: #eceef1;
+}
+
+* {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+a {
+  margin: 0 auto;
+  text-decoration: none;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.frame {
+  max-width: 349px;
+  border-radius: 4px;
+  margin: auto;
+  background: white;
+  box-sizing: border-box;
+  color: black;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  font-size: 14px;
+  font-weight: normal;
+  border: 1px solid rgba(0, 0, 0, 0.3);
+  padding: 25px;
+}
+
+h2 {
+  text-align: center;
```

Comment:
- [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

Please apply DRY CSS rules to prevent repeating the same rules.